docs/openapi/components/schemas/common/EDDSHP.yml

```diff
@@ -0,0 +1,397 @@
+    type: number
+    $linkedData:
+      term: grossAreaAcres
+      '@id': http://rs.tdwg.org/dwc/terms/measurementValue
+  percentCover:
+    title: Percent Cover
+    description: Percent cover of invasive species as decimal (e.g. 0.5).
+    type: number
+    $linkedData:
+      term: percentCover
+      '@id': http://rs.tdwg.org/dwc/terms/measurementValue
+  density:
+    title: Density
+    description: Range of percent cover or description (low, medium, common, rare, etc.).
+    type: string
+    $linkedData:
+      term: density
+      '@id': http://rs.tdwg.org/dwc/terms/measurementRemarks
+  quantity:
+    title: Quantity
+    description: Exact number of the observed subject or number of biocontrol agents released.
+    type: number
+    $linkedData:
+      term: quantity
+      '@id': http://rs.tdwg.org/dwc/terms/organismQuantity
+  quantityUnits:
+    title: Quantity Units
+    description: Unit of measure (individuals, adults, grams, etc.).
+    type: string
+    $linkedData:
+      term: quantityUnits
+      '@id': http://rs.tdwg.org/dwc/terms/organismQuantityType
+  approximateQuantity:
+    title: Approximate Quantity
+    description: Approximate or range of subjects observed or biocontrol agents released.
+    type: string
+    $linkedData:
+      term: approximateQuantity
+      '@id': http://rs.tdwg.org/dwc/terms/organismQuantity
+  incidence:
+    title: Incidence
+    description: Number of measure for disease occurrence on subject population.
+    type: string
+    $linkedData:
+      term: incidence
+      '@id': http://rs.tdwg.org/dwc/terms/measurementValue
+  severity:
+    title: Severity
+    description: Number of measure for disease impact on individual or sampled subject.
+    type: string
+    $linkedData:
+      term: severity
+      '@id': http://rs.tdwg.org/dwc/terms/measurementValue
+  managementStatus:
+    title: Management Status
+    description: Stage of management activities on the subject.
+    type: string
+    $linkedData:
+      term: managementStatus
+      '@id': https://schema.org/status
+  habitat:
+    title: Habitat
+    description: Ecosystem where the subject was located.
+    type: string
+    $linkedData:
+      term: habitat
+      '@id': http://rs.tdwg.org/dwc/terms/habitat
+  siteName:
+    title: Site Name
+    description: Specific name of area by organization.
+    type: string
+    $linkedData:
+      term: siteName
+      '@id': http://rs.tdwg.org/dwc/terms/locationID
+  recordBasis:
+    title: Record Basis
+    description: The basis of the occurrence.
+    type: string
+    $linkedData:
+      term: recordBasis
+      '@id': http://rs.tdwg.org/dwc/terms/samplingProtocol
+  surveyor:
+    title: Surveyor
+    description: Name or initials of surveyor other than Reporter.
+    type: string
+    $linkedData:
+      term: surveyor
+      '@id': http://rs.tdwg.org/dwc/terms/recordedBy
+  dateUncertaintyDays:
+    title: Date Uncertainty in Days
+    description: Date Accuracy in Days.
```

Comment:
Description should clarify title, not make it less clear. "Uncertainty" and "Accuracy" are roughly contradictory...